### ISSUE TITLE
Include visible fields from superclass 

### ISSUE THREAD

**Issue by Rapster** [helospark/SparkBuilderGenerator#35]
From the documentation:

> Include visible fields from superclass 
> Whether to also include visible fields from super class. Field is visible if it's public, protected. It's also visible if it has a default scope and it's in the same package. Private fields are never visible, therefore not set.

It is also visible if the superclass contains a public getter on the private field. I have a superclass with private  fields and public getters. I expect to be able to generate a builder with the superclass fields

**Comment by helospark** [comment]
Hi @Rapster,

Thanks for opening this.
Just to clarify you have a superclass that has fields with setters and you would like to include these fields in the builder?
Or do you have getters? How would the fields get set?

I'm think about the following, please check if this is what you had it in mind.

Given this superclass:

```java
public class SuperClass {
	private String data;

	public String getData() {
	    return data;
	}

	public void setData(String data) {
	    this.data = data;
	}
}
```

and this child class:

```java
public class ChildClass extends SuperClass {
	private String childField;
}
```

the following builder would be the output:

```java
public class ChildClass extends SuperClass {
	private String childField;

	private ChildClass(Builder builder) {
	    this.childField = builder.childField;
	    super.setData(builder.data);
	}

	public static Builder builder() {
	    return new Builder();
	}

	public static final class Builder {
	    private String data;
	    private String childField;

	    private Builder() {
	    }

	    public Builder withChildField(String childField) {
	        this.childField = childField;
	        return this;
	    }

	    public Builder withData(String data) {
	        this.data = data;
	        return this;
	    }

	    public ChildClass build() {
	        return new ChildClass(this);
	    }
	}
}
```

**Comment by Rapster** [comment]
My bad, setter and not getter. Your example is definetely what I had in mind :wink:

**Comment by helospark** [comment]
Hey @Rapster,
I have released version 0.0.15, please try to upgrade and check if everything is working as expected.

**Comment by Rapster** [comment]
Great I'll let know asap ;-)
